### PR TITLE
Use example tracts in the notebook

### LINKED ISSUE #8: Update the tracts used in notebooks and related instructions
In the tutorial notebooks, we should be more explicit about which tracts are used in the notebook, and maybe also what would be a minimal set of tracts for the notebook to work at all.

We should also update the tracts used in the examples to the subset of 3830, 3831, 4028, 4029, which will be defined as an "example subset" in the download portal. 

Thanks to @JoanneBogart for the suggestion.

### PR DESCRIPTION
**Pull request by yymao**
This PR updates the example notebooks to use the example tract set: 3830, 3831, 4028, 4029. 

In the truth notebook, I also added a note about cosmoDC2 as a way to advertise it for those who are interested. 

This PR fixes #8.

### CODE REVIEW

File: notebooks/dc2/object_table_gcrcatalogs_intro.ipynb [hunk 1]
```diff
@@ -18,14 +18,18 @@
...
     "\n",
     "Make sure you have followed the instructions on the [DESC Data Portal](https://lsstdesc-portal.nersc.gov/) to \n",
-    "download the data files, install `GCRCatalogs`, and set up `root_dir` for `GCRCatalogs`."
-   ],
-   "cell_type": "markdown",
-   "metadata": {}
+    "download the data files, install `GCRCatalogs`, and set up `root_dir` for `GCRCatalogs`.\n",
+    "\n",
+    "In this example notebook we will use up to 4 tracts: 3830, 3831, 4028, 4029. \n",
+    "If you have downloaded the example set from the Data Portal, it has included these four tracts, so you are all set. \n",
```
Comment (JoanneBogart) [inline notebooks/dc2/object_table_gcrcatalogs_intro.ipynb:30]:
```suggestion
"The example set from the Data Portal includes these four tracts, so If you have downloaded it you are all set. \n",
```

File: notebooks/dc2/truth_match_table.ipynb [hunk 2]
```diff
@@ -19,14 +19,18 @@
...
     "\n",
     "Make sure you have followed the instructions on the [DESC Data Portal](https://lsstdesc-portal.nersc.gov/) to \n",
-    "download the data files, install `GCRCatalogs`, and set up `root_dir` for `GCRCatalogs`."
-   ],
-   "cell_type": "markdown",
-   "metadata": {}
+    "download the data files, install `GCRCatalogs`, and set up `root_dir` for `GCRCatalogs`.\n",
+    "\n",
+    "In this example notebook we will only use tract 3830. \n",
+    "If you have downloaded the example set from the Data Portal, it has included this tract, so you are all set. \n",
```
Comment (JoanneBogart) [inline notebooks/dc2/truth_match_table.ipynb:31]:
```suggestion
    "The example set from the Data Portal includes this tract, so if you have downloaded that set you are ready to go. \n" 
```

**Review by JoanneBogart — CHANGES_REQUESTED** [review]
I made a couple minor suggestions on rewording.  
When we discussed this before I had suggested early on including a line something like
SAMPLE_TRACTS = [3830, 3831, 4028, 4029]
and then using it consistently.  Perhaps there should also be a designated single sample tract.   I think it would make the notebook more readable and also easier to maintain. Is there a reason not to make this change?

**Comment by yymao** [comment]
It was mostly to also demonstrate how `tract_filter` works. If people want to copy code from a cell for their analysis it seems better to have tract number spelled out.  Otherwise we could define something like 

```
one_tract_example = tract_filter(3830)
four_tract_example = tract_filter([3830, 3831, 4028, 4029]) 
```

and use them in `native_filters` directly.

**Review by JoanneBogart — APPROVED** [review]
Looks good. Thanks @yymao

**Comment by yymao** [comment]
Thanks!

**Pull request merged**